### DISCUSSION TITLE
Number of items in bibliography

### DISCUSSION THREAD

**Discussion by amrbz** [typst/typst#4344] [General]
Is there any way to count the total number of the displayed items in bibliography?

**Comment by jamesrswift** [comment]
https://discordapp.com/channels/1054443721975922748/1247213680848932906/1247221094583242762

**Reply by amrbz** [reply]
Can’t have access to the conversation 🙄

**Reply by jamesrswift** [reply]
Ah sorry.

From jimpjorps:
```
#context query(selector(ref)).filter(it => it.element == none).map(it => it.target).dedup().len()
```
playing around with context, this is what I come up with as a working selector

**Reply by amrbz** [reply]
Worked for me, as well. Thanks )
